nixos/doc/manual/installation/installing-from-other-distro.section.md

```diff
@@ -160,6 +160,12 @@ The first steps to all these are the same:
     Refer to the `nixos-generate-config` step in
     [](#sec-installation) for more information.
 
+    ::: {.note}
+    The systemd initrd (`boot.initrd.systemd.enable`) cannot be used
```

Comment:
There's an `option` markup that will eventually allow us to link to reference documentation for that option. Please use that.

Reply:
Also, initrd is a very obscure term for beginners, please link to a definition/explanation.

Reply:
I assume you're referring to `[](#opt-boot.initrd.systemd.enable)` when you talk about the `option` markup? I couldn't find an example of anything else like that in use.